### PR TITLE
Wk 213 navigate on save

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/components/locationForm/LocationForm.jsx [hunk 1]
```diff
@@ -11,26 +12,38 @@ import FormInput from './components/FormInput';
 import FileInput from './components/FileInput';
 import fromFeature, { toFormData, toFeature } from './services/normalize';
 import { sendData } from 'shared/api/api';
+import { appRoutes } from '../../constants';
 
-const LocationForm = ({ id }) => {
+const LocationForm = withRouter(({ id, history }) => {
```
Comment (janjaap) [inline src/components/locationForm/LocationForm.jsx]: You can use `useHistory` hook instead

File: src/components/locationForm/LocationForm.jsx [hunk 2]
```diff
@@ -43,14 +56,23 @@ const LocationForm = ({ id }) => {
...
+  }, [register, id]);
 
   return (
     <>
-      <form onSubmit={handleSubmit(onSubmit)} action="" noValidate>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        action=""
+        noValidate
+        onReset={onReset}
```
Comment (janjaap) [inline src/components/locationForm/LocationForm.jsx]: Doesn't seem to be supported by IE: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onreset

File: src/components/map/hooks/useMap.js [hunk 3]
```diff
@@ -19,13 +19,13 @@ const useMap = () => {
...
 
     // Add the stadsdelen WMS
-    L.tileLayer
-      .wms('https://map.data.amsterdam.nl/maps/gebieden?', {
-        layers: ['stadsdeel'],
-        transparent: true,
-        format: 'image/png',
-      })
-      .addTo(mapRef.current);
+    // L.tileLayer
```
Comment (janjaap) [inline src/components/map/hooks/useMap.js]: Remove commented out lines?

File: src/test/utils.js [hunk 4]
```diff
@@ -15,6 +15,11 @@ export const withUserContext = (Component, user, route) => {
   );
 };
 
-export const withTheme = Component => (
-  <ThemeProvider>{Component}</ThemeProvider>
-);
+export const withTheme = (Component, route) => {
+  if (route) history.push(route);
```
Comment (janjaap) [inline src/test/utils.js]: Doesn't this add too much responsibility to this simple util function?

File: src/components/locationForm/LocationForm.jsx (hunk 1 above)
Comment (janjaap) [reply src/components/locationForm/LocationForm.jsx]: You can also use `useLocation`

File: src/components/locationForm/LocationForm.jsx (hunk 2 above)
Comment (aditudorache) [reply src/components/locationForm/LocationForm.jsx]: thanks, changed.

File: src/components/map/hooks/useMap.js (hunk 3 above)
Comment (aditudorache) [reply src/components/map/hooks/useMap.js]: thanks

File: src/test/utils.js (hunk 4 above)
Comment (aditudorache) [reply src/test/utils.js]: moved to withHistory hoc

File: src/components/locationForm/LocationForm.jsx (hunk 1 above)
Comment (aditudorache) [reply src/components/locationForm/LocationForm.jsx]: thanks!!

**Pull request merged**